galgebra/printer.py

```diff
@@ -983,20 +1100,27 @@ def Format(Fmode=True, Dmode=True, dop=1, inverse='full'):
     GaLatexPrinter.Fmode = Fmode
     GaLatexPrinter.inv_trig_style = inverse
 
-    if Format_cnt == 0:
+    if Format_cnt == 0:  # Only execute first time Format is called
         Format_cnt += 1
 
         GaLatexPrinter.dop = dop
         GaLatexPrinter.latex_flg = True
+        # Overload python 3 print function
         GaLatexPrinter.redirect()
 
         Basic.__str__ = lambda self: GaLatexPrinter().doprint(self)
         Matrix.__str__ = lambda self: GaLatexPrinter().doprint(self)
         Basic.__repr__ = lambda self: GaLatexPrinter().doprint(self)
         Matrix.__repr__ = lambda self: GaLatexPrinter().doprint(self)
 
-        if isinteractive():
+        if isinteractive():  # Set up for Jupyter Notebook/Lab
             init_printing(use_latex= 'mathjax')
+            from IPython.core.interactiveshell import InteractiveShell
+            #  Allow multiple outputs in an output cell
+            #  https://forums.fast.ai/t/jupyter-notebook-enhancements-tips-and-tricks/17064/2
+            InteractiveShell.ast_node_interactivity = "all"
```

Comment:
This isn't about allowing multiple outputs, it's about producing them automatically.
The user can always print multiple outputs with `display(...)` if they want to.

This doesn't seem like a good idea to me. If users want it, they can do it themselves.